content/operate/kubernetes/reference/supported_k8s_distributions.md

```diff
@@ -12,6 +12,8 @@ weight: 30
 
 Each release of Redis Enterprise for Kubernetes is thoroughly tested against a set of Kubernetes distributions. The table below lists Redis Enterprise for Kubernetes versions and the Kubernetes distributions they support.
 
+{{<note>}}Testing is done with the default container runtime for each platform. Other container runtimes are not tested or certified.”{{</note>}}
```

Comment:
Maybe we can generalize this note and say something like "Each platform is tested with its default configuration, including storage, network, security, and container runtime components".
WDYT? Also a question for @bascar about how we position our platforms support.